modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -81,6 +80,8 @@ ParallelCoordinates::ParallelCoordinates()
     , axisProperties_("axisProps_", "Axis")
     , colors_("colors", "Colors")
     , axisColor_("axisColor", "Axis Color", vec4(.6f, .6f, .6f, 1))
+    , axisHoverColor_("axisHoverColor", "Axis Hover Color", vec4(.3f, .3f, .3f, 1))
+    , axisSelectedColor_("axisSelectedColor", "Axis Selected Color", vec4(.8f, .0f, .0f, 1))
```

Comment:
Red color as default? How about a brighter color than the other axis

Reply:
Maybe fully white for selected, light gray for hover and dark gray by default?
<img width="423" alt="pcp" src="https://user-images.githubusercontent.com/8807821/52802819-fcfe9280-3080-11e9-83ef-ceb1152a2d09.png">

Reply:
Probably not fully white. A bit brighter than hover.

Reply:
![image](https://user-images.githubusercontent.com/8807821/52844923-1138a300-3106-11e9-8774-40599ec6a3e2.png)

Better? 🥇

Reply:
The selected axis is also a little bit wider than the rest.